packages/react-native-editor/index.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * Internal dependencies
+ */
+import './src';
```

Comment:
You can set `main` to `src/index` in `package.json` and remove this file.

Reply:
I think [metro](https://github.com/facebook/metro) still needs it when bundling.

Reply:
Sounds good, I missed that fact :)